pkg/admissionpolicy/testutils/util.go

```diff
@@ -176,6 +175,8 @@ func EnvTestWithAuditPolicy(policyYaml string, env *envtest.Environment) {
 //
 // This is intended to allow for loading the admission policies into envtest,
 // therefore it doesn't return errors, but Expects() them not to happen.
+//
+//nolint:gocognit,funlen
```

Comment:
Yay 😂 

Non blocking thought: should we be excluding these util /test utils from linting?